packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -789,7 +796,41 @@ describe('plugin-meetings', () => {
               meeting: sinon.match.instanceOf(Meeting), type: 'test meeting added type'
             });
           });
+
+          it('creates the meeting from a successful meeting info fetch with random delay', async () => {
+            const nowTimeStamp = Date.now();
+            const FAKE_TIME_TO_START = 0.1 * 60 * 1000;
+            const fakeMeetingStartTimeStamp = nowTimeStamp + FAKE_TIME_TO_START;
+            const fakeMeetingStartTimeDate = new Date(fakeMeetingStartTimeStamp);
+            const fakeMeetingStartTimeString = fakeMeetingStartTimeDate.toISOString();
+            const fakeDestination = {
+              meeting: {
+                startTime: fakeMeetingStartTimeString
+              },
+              fullState: {
+                active: false
+              }
+            };
+
+            const meeting = await webex.meetings.createMeeting(fakeDestination, 'test type', true);
+
+            assert.instanceOf(meeting, Meeting, 'createMeeting should eventually resolve to a Meeting Object');
+            assert.notCalled(webex.meetings.meetingInfo.fetchMeetingInfo);
+            assert.calledOnce(setTimeoutSpy);
+            clock.tick(FAKE_TIME_TO_START);
+            assert.calledOnce(webex.meetings.meetingInfo.fetchMeetingInfo);
+            assert.calledOnce(MeetingsUtil.getMeetingAddedType);
+            assert.calledTwice(TriggerProxy.trigger);
+            assert.calledWith(webex.meetings.meetingInfo.fetchMeetingInfo, fakeDestination, 'test type');
+            assert.calledWith(MeetingsUtil.getMeetingAddedType, 'test type');
+            assert.calledWith(TriggerProxy.trigger, sinon.match.instanceOf(Meetings), {
```

Comment:
this notification should have been sent already before the timer expiry, right? so we should move that check before the call to `clock.tick()` and here instead we should check the "meeting:meetingInfoAvailable" event being sent

Reply:
I still don't see the check for "meeting:meetingInfoAvailable" event - it's not tested anywhere